ApolloExtra.lua

```diff
@@ -2418,48 +2473,115 @@ OlympusLootData.ApolloUpgrade = {
+				RequiredTextLines = { "ApolloAboutDaughters01" },
+				{ Cue = "/VO/Apollo_0063",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "Her name Cephisso and she's probably my most shy daughter of the three. She's very quiet... like very... Sometimes, she speaks and I didn't she was in the room... So, for a quiet lifestyle, she could be the one for you!" },
+			},
+			ApolloAboutDaughters04 =
+			{
+                Name = "ApolloAboutDaughters04",
+				PlayOnce = true,
+				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				RequiredTextLines = { "ApolloAboutDaughters01" },
+				{ Cue = "/VO/Apollo_0064",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "Borysthenis is the strong one, she's not nervous. She can even move mountains! Just be careful with her hugs, she cracked my back one time and I still fell soar for it, haha. But hey, if that's not showing love, what is it?! If you want someone to get your back, she's the one!" },
+			},
+			ApolloAboutDaughters05 =
+			{
+                Name = "ApolloAboutDaughters05",
+				PlayOnce = true,
+				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				EndCue = "/VO/ZagreusHome_3760",
+				RequiredTextLines = { "ApolloAboutDaughters02", "ApolloAboutDaughters03", "ApolloAboutDaughters04" },
+				{ Cue = "/VO/Apollo_0065",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "Well, I have talked to my daughters, and they kind of don't want, hmm, a distant relationship, and since I don't see you going anywhere, I think they'll end this here." },
+			},
+			ApolloAboutHydra =
+			{
+                Name = "ApolloAboutHydra",
+				PlayOnce = true,
+				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				RequiredBiome = "Asphodel",
+				RequiredAnyRoomsLastRun = { "B_Boss01", "B_Boss02" },
 				RequiredTextLines = { "ZeusFirstPickUp", "PoseidonFirstPickUp", "AthenaFirstPickUp", "AphroditeFirstPickUp", "AresFirstPickUp", "ArtemisFirstPickUp", "DionysusFirstPickUp", "ApolloGift01", "ApolloFirstPickUp" },
-				RequiredGodLoot = "AresUpgrade",
-				{ Cue = "/VO/Apollo_0046",
+				{ Cue = "/VO/Apollo_0165",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "I sense the mark of my Lord Ares on you, love. He's poorly understood among the gods, yet surely you can see fair qualities in him, and me?" },
+					Text = "Hydra has snake's head, right? um, I hate snakes, specially pythons, they are the worst. Still, I got the upper hand against them, here that should help." },
```

Comment:
"Hydra has snakes for heads, right? um, I hate snakes, especially pythons, they are the worst. Still, I got the upper hand against them. Here, that should help."